### DISCUSSION TITLE
Use with XCode Cloud?

### DISCUSSION THREAD

**Discussion by paulgessinger** [liamnichols/xcstrings-tool#42] [Q&A] [answered]
Has anyone tried / gotten this to work with XCode Cloud? I've been running into issues, presumably because the cloud build won't run the plugin.

<img width="1040" alt="image" src="https://github.com/liamnichols/xcstrings-tool/assets/1058585/0977c2c0-8a9c-40f4-ab47-9aac269c6013">

**Comment by liamnichols** [comment, marked as answer]
Hey @paulgessinger.. Yes @hiragram did and he fixed the issue in #36 but I haven't released a new version yet! 

I was hoping to fix some other bugs first, but that never happened 😅 

I'll release a new version this afternoon 👍

**Reply by liamnichols** [reply]
I have now released 0.1.2 (+ the precompiled executable for the plugin) so you should be able to give this a try. 

Let me know if there are any issues 👍

**Reply by paulgessinger** [reply]
This unfortunately didn't change anything for me. :/

**Reply by hiragram** [reply]
@paulgessinger You may need `defaults write com.apple.dt.Xcode IDESkipPackagePluginFingerprintValidatation -bool YES` in `ci_post_clone.sh`

**Reply by paulgessinger** [reply]
@hiragram that helpes with the build, thanks!
